nautobot/extras/tests/test_filters.py

```diff
@@ -248,44 +257,66 @@ def test_role(self):
         )
 
     def test_type(self):
-        device_types = self.device_types[:2]
-        params = {"device_type_id": [device_types[0].pk, device_types[1].pk]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), len(device_types))
-        params = {"device_type": [device_types[0].slug, device_types[1].slug]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), len(device_types))
+        device_types = list(self.device_types[:2])
+        filter_params = [
+            {"device_type_id": [device_types[0].pk, device_types[1].pk]},
+            {"device_type": [device_types[0].pk, device_types[1].slug]},
```

Comment:
These should certainly be changed over in `next` to use `generic_filter_tests` at some point but that's beyond the scope of this PR.